README.md

```
@@ -18,6 +18,32 @@ Note that installing the operator using this method requires adding the openshif
+The controller validates the scheduler image set in `numaresourcesscheduler.spec.image` against an embedded set of valid images by their digests. This set is fetched from the production registry. To pass the image validation, the following must be true:
+1. The image is pinned by SHA digest.
+2. The digest belongs to an image that's available in the production registry `registry.redhat.io` and is of the same minor operator version or is the latest of the previous minor version. 
+
+If a privileged user wants to extend the allow list to include other digests, that is possible by providing a comma-separated list of SHA digests as an environment variable `SCHEDULER_IMAGE_DIGESTS` under `subscription.spec.config.env`.
+
+If the privileged user wants to disable image validation, set `DISABLE_SCHEDULER_IMAGE_VALIDATION` to `true` under `subscription.spec.config.env`.
+
+
+On locally installed operators (no OLM), one can directly set the environment variables through the manager deployment using this command:
+
+```
```

markdownlint-cli2 (0.22.1)

[warning] Fenced code blocks should have a language specified

(MD040, fenced-code-language)